etc/twcmanager/config.json

```diff
@@ -374,7 +374,9 @@
         # a pi, it may mean considering alternatives such as a remote database
         # to reduce write cycles.
         "Console": {
-            "enabled": true
+            "enabled": true,
+            # simple logging format omits timestamp and colorization
+            "simple": false
```

Comment:
I might consider commenting out this line and setting it to true, since the default is already "false".  Then someone can just uncomment it to get the simple behavior.

Reply:
Good point, I pushed an according change.